package.json

```diff
@@ -33,7 +36,7 @@
     "eslint-config-prettier": "5.0.0",
     "eslint-plugin-import": "2.14.0",
     "eslint-plugin-jest": "22.7.0",
-    "eslint-plugin-jest-formatting": "^0.0.10",
+    "eslint-plugin-jest-formatting": "file:.",
```

Comment:
This uses the current on-the-filesystem version of the package which was helpful to me during development, but definitely not required.